routes/api/users.js

```diff
@@ -37,4 +37,17 @@ router.post(
   userControllers.logout
 );
 
+// User functionality
+router.get(
+  '/users/authors',
+  passport.authenticate('jwt', { session: false }),
+  userControllers.getAllAuthors
+);
+router.get(
```

Comment:
I think authentication should be optional as we want to view author information only and this functionality should be available to all users even if they are not authenticated.
Check the [readme](https://github.com/andela/ah-kgl-avengers-backend#get-profile) file of the project.